### DISCUSSION TITLE
Indistinguishable YAML syntax highlighting colors in protan and deutan themes

### DISCUSSION THREAD

**Discussion by ghost** [community/community#16254] [Other Feature Feedback, Questions, & Ideas]
### Dark
![default dark](https://user-images.githubusercontent.com/104722305/167444509-776da8fc-70aa-4e37-8fcd-8323eaa72154.png)
_Default_

![protanopia & deuteranopia dark](https://user-images.githubusercontent.com/104722305/167444516-d135cc3b-7d5e-418e-bf5d-7349923d2c1e.png)
_Protanopia & Deuteranopia_

### Light
![default light](https://user-images.githubusercontent.com/104722305/167444513-abe30524-e0f8-4a36-aee9-19c1ca77bd01.png)
_Default_

![protanopia & deuteranopia light](https://user-images.githubusercontent.com/104722305/167444514-297ee8af-155d-4ad6-bec7-e734f5956227.png)
_Protanopia & Deuteranopia_

**Comment by Amejonah1200** [comment]
I can confirm this, even if I'm not colorblind.
